Cargo.lock

```diff
@@ -89,6 +89,52 @@ version = "1.1.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "d468802bab17cbc0cc575e9b053f41e72aa36bfa6b7f55e3529ffa43161b97fa"
 
+[[package]]
+name = "axum"
```

Comment:
is this required?

Reply:
it will be when we add the outbound API to the mock controller. using `tonic`'s "transport" feature is necessary to serve multiple gRPC APIs on the same port, and "transport" depends on `axum`, because it uses `axum`'s request router to implement this. 

we could avoid the dependency in this PR and only add it in the subsequent PR that adds the outbound API to the mock controller. however, i decided not to do that, because this would make the diff in the subsequent PR bigger --- it would change how the server is started. this way, we would just add one line to add the outbound API service to the server builder, which i thought was a neater diff. up to you though.

Reply:
Do we need all of axum's transitive dependencies? It's not a blocker, but it's not awesome to have to take on a bunch of new deps for what boils down to a match statement.

Reply:
i can replace this with a handwritten router if that's preferable!

Reply:
It depends, but if we're in the ballpark of <100LOC, i'd say that's preferable to the `transport` feature. Same goes for the policy controller, for that matter.

Reply:
Y'know what... we can always tear it out later if it's a pain. #FISI.